static/src/stylesheets/module/facia/_items.scss

```diff
@@ -319,3 +319,10 @@
         margin-right: 0;
     }
 }
+
+// Temporary hack, as MPU candidate is broken on tablet
+.fc-slice--t-t-mpu .fc-slice__item--mpu-candidate {
+    @include mq(tablet, desktop) {
+        display: none;
+    }
+}
```

Comment:
careful, as I think this will still make the ad call. If you hide the actual ad slot, that will stop the call from being made